### DISCUSSION TITLE
How do I include third side markdown inside .ipynb notebooks?

### DISCUSSION THREAD

**Discussion by atdiausx2** [quarto-dev/quarto-cli#7874] [Q&A]
### Description

The documentation hints that there should be already a way to include text from markdown files inside .ipynb notebooks to render it entirely , but if in a markdown (or raw) .ipynb cell I write

```
{{< includes ./_abstract.md >}} 

``` 
, this does not affect my PDF preview anyhow, nor it is in .tex.

**Comment by cscheid** [comment]
it should be `include`, not `includes`.

**Reply by atdiausx2** [reply]
Thank you for pinpointing this ridiculous typo. This is however not the essence of the question. 
Of course I can make an include in `.qmd` file and it works, but I work in VSC and I usually use  only `.ipynb` files well since `.qmd` with python under complex environment causes slightly more issues and shows all results aside not how they go. 

The question is how in what cell should I type this include so that it would actually include the text in `.ipynb` file. Thanks.

**Reply by cscheid** [reply]
The following works for me:

## 7874.ipynb
````
{
  "cells": [
    {
      "cell_type": "raw",
      "metadata": {},
      "source": [
        "---\n",
        "title: test\n",
        "format: html\n",
        "---"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "{{< include _subtest.md >}}"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {},
      "source": [
        "print(\"Hello, world\")"
      ],
      "execution_count": null,
      "outputs": []
    }
  ],
  "metadata": {
    "kernelspec": {
      "name": "python3",
      "language": "python",
      "display_name": "Python 3 (ipykernel)"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 4
}
````

## _subtest.md

```md
This is some markdown content.

This is some more markdown.
```

# Rendering

```
$ quarto render 7874.ipynb --to html --execute

Executing '7874.ipynb'
  Cell 1/1: ''...Done

pandoc
  to: html
  output-file: 7874.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png

metadata
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  title: test

Output created: 7874.html
```

That produces this output:

<img width="823" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/9537279b-1340-4d9c-a2c0-640c8f523f5f">

It's possible that you're experiencing a different issue of some kind. This is why ask folks for a simple, reproducible example.
